wom/src/main/scala/wdl4s/wdl/WdlNamespace.scala

```diff
@@ -419,9 +419,16 @@ object WdlNamespace {
     expression.variableReferences collect { case variable if container.resolveVariable(variable.terminal.sourceString).isEmpty => variable.terminal }
 
   private def validateDeclaration(declaration: DeclarationInterface, wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): Seq[SyntaxError] = {
+    // Used to filter task declaration nodes whose parents are subworkflow calls. Subworkflows will cause task declarations to appear twice among
+    // the namespace's descendants: once with a parent that is the containing static workflow and again with a parent that is the dynamic workflow
+    // call. Looking for variable references when the parent is a workflow call will fail if the variable references a call output since only
+    // declarations are made children of workflow calls. This variable reference check is also redundant to the check that will happen with the
+    // copy of the declaration that has a static workflow parent.
```

Comment:
This look like it should be a single javadoc style comment since it should be considered attached to the `parentIsNotAWorkflowCall ` method

Reply:
This is an internal function and wouldn't render as Scaladoc, but I could certainly use the multi-line syntax if that was deemed more palatable.

Reply:
Despite my previous thumb I'll chime in to say that I concur in that I prefer 

     /*
        foo
     */

for multiline comments. I realize that wasn't a multiline comment :)